src/common/snippets/src/generator.cpp

```diff
@@ -34,7 +34,8 @@ Generator::LoweringResult Generator::generate(lowered::LinearIR& linear_ir, cons
     OV_ITT_TASK_NEXT(GENERATE, "::EmitCode")
     auto loops2DKernel = std::make_shared<op::Kernel>(linear_ir);
     loops2DKernel->compile_params = compile_params;
-    std::shared_ptr<Emitter> kernel = target->get(op::Kernel::get_type_info_static())(loops2DKernel);
+    auto loops2DKernelExpr = linear_ir.create_expression(loops2DKernel, std::vector<lowered::PortConnectorPtr>{});
```

Comment:
Note: this invalidates our LIR since we won't be able to reuse it again (e.g. in dynamic pipeline). Let's leave at as is right now, but we'll have to deal with it in future (probably use constructor from the Op, will see).